app/uk/gov/hmrc/helptosavefrontend/views/closeaccount/close_account_are_you_sure.scala.html

```diff
@@ -77,27 +80,25 @@ <h1>@title</h1>
     }
 
 
-    <p>@messages("hts.close-account.p1")</p>
-    <p>@Html(Messages("hts.close-account.p2"))</p>
+    <p class="govuk-body">@messages("hts.close-account.p1")</p>
+    <p class="govuk-body">@Html(Messages("hts.close-account.p2"))</p>
 
-    <div class="panel panel-border-narrow">
-        <p>
-            @messages("hts.close-account.contact.telephone")<br/>
-            <strong class="bold">@messages("hts.close-account.contact.telephone.number")</strong>
-        </p>
-        <p>
-            @messages("hts.close-account.contact.outside-uk")<br/>
-            <strong class="bold">@messages("hts.close-account.contact.outside-uk.number")</strong>
-        </p>
-    </div>
+@viewHelpers.govukInsetText(InsetText(
+    content = HtmlContent("<p class=\"govuk-body\">"+s"${messages("hts.close-account.contact.telephone")}" +
+            "<br/><strong>"+s"${messages("hts.close-account.contact.telephone.number")}</strong></p>" +
+            "<p class=\"govuk-body\">"+s"${messages("hts.close-account.contact.outside-uk")}" +
+            "<br/><strong>"+s"${messages("hts.close-account.contact.outside-uk.number")}</strong></p>"
+    )
+))
 
-    <h2 class="heading-small">@messages("hts.close-account.h2")</h2>
-    <p>@messages("hts.close-account.p3")</p>
-    <p>@messages("hts.close-account.p4")</p>
+    <h2 class="govuk-heading-s">@messages("hts.close-account.h2")</h2>
+    <p class="govuk-body">@messages("hts.close-account.p3")</p>
+    <p class="govuk-body">@messages("hts.close-account.p4")</p>
 
-    <a href=@{appConfig.nsiManageAccountUrl}
-    data-ga-event-category="link - click"
-    data-ga-event-action="Are you sure you want to close your account"
-    data-ga-event-label="Back">@messages("hts.close-account.back")</a>
+@viewHelpers.govukBackLink(BackLink(
```

Comment:
pretty sure this should not be back link - back links are only used in layout and not in main content and have different styling to regular links